cluster-autoscaler/metrics/metrics.go

```diff
@@ -50,10 +54,19 @@ const (
 	LogLongDurationThreshold = 5 * time.Second
 )
 
+// Names of Cluster Autoscaler operations
 const (
-	ScaleDownNodeDeletion      = "scaleDown:nodeDeletion"
-	ScaleDownFindNodesToRemove = "scaleDown:findNodesToRemove"
-	ScaleDownMiscOperations    = "scaleDown:miscOperations"
+	ScaleDown                  FunctionLabel = "scaleDown"
+	ScaleDownNodeDeletion      FunctionLabel = "scaleDown:nodeDeletion"
+	ScaleDownFindNodesToRemove FunctionLabel = "scaleDown:findNodesToRemove"
+	ScaleDownMiscOperations    FunctionLabel = "scaleDown:miscOperations"
+	ScaleUp                    FunctionLabel = "scaleUp"
+	FindUnneeded               FunctionLabel = "findUnneeded"
+	UpdateClusterStateLabel    FunctionLabel = "updateClusterState"
```

Comment:
It clashes with a function name in this file, but I don't want to put the Label suffix on all the names. Couldn't think of anything clever.